scss/_utilities/_variables.scss

```diff
@@ -74,18 +74,22 @@ $weight-sbold: 600;
 $weight-bold: 700;
 
 // Typography
-$font-hero: 54px;
-$font-larger: 36px;
-$font-large: 32px;
-$font-medium: 24px;
-$font-regular: 18px; // base font size
-$font-small: 16px;
-$font-smaller: 13px;
+$font-gigantic: 83.989px;
```

Comment:
It definitely makes stylesheets a lot uglier, but I _think_ browsers should just round them to the nearest pixel value so it's not a huge deal I guess.